_includes/nav.html

```diff
@@ -8,7 +8,7 @@
             {% if page.parent == node.title or page.grand_parent == node.title %}
               {% assign first_level_url = node.url | absolute_url %}
             {% endif %}
-            <a href="{{ node.url | absolute_url }}" class="navigation-list-link{% if page.url == node.url %} active{% endif %}">{{ node.title }}</a>
+            <a href="{{ node.url | absolute_url }}" class="navigation-list-link{% if page.url == node.url %} active{% endif %}">{{ node.sidebar_label || node.title }}</a>
```

Comment:
I think this should be done with a default filter instead

Reply:
@thealjey could you add a [suggested commit here](https://haacked.com/archive/2019/06/03/suggested-changes/)?